core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -18,71 +18,77 @@
-        </ul>
-      </li>
-      <li ng-if="$ctrl.CLASSROOM_PAGE_IS_SHOWN"
-          ng-class="{'open' : $ctrl.activeMenuName === 'classroomMenu'}"
-          uib-dropdown
-          ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_CLASSROOM"
-          aria-hidden="false"
-          class="nav-item  oppia-navbar-clickable-dropdown protractor-test-classroom-oppia-list-item">
-        <a ng-keydown="$ctrl.onMenuKeypress($event, 'classroomMenu', {shiftTab: $ctrl.ACTION_CLOSE, enter: $ctrl.ACTION_OPEN})" class="nav-link oppia-navbar-tab dropdown-toggle"
-           role="menuitem"
-           aria-haspopup="true"
-           aria-expanded="false"
-           aria-hidden="false"
-           tabindex="0"
-           ng-mouseover="$ctrl.openSubmenu($event, 'classroomMenu')"
-           ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)">
-          <span class="oppia-navbar-tab-content" translate="I18N_TOPNAV_CLASSROOM"></span>
-        </a>
-        <ul uib-dropdown-menu class="uib-dropdown-menu oppia-navbar-dropdown"
-            ng-mouseover="$ctrl.openSubmenu($event, 'classroomMenu')"
-            ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)"
-            role="menu">
-          <li class="nav-item"> <a ng-keydown="$ctrl.onMenuKeypress($event, 'classroomMenu', {shiftTab: $ctrl.ACTION_CLOSE})" class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link" href="/math" translate="I18N_TOPNAV_CLASSROOM_BASIC_MATHS"></a></li>
-        </ul>
-      </li>
-      <li class="nav-item" ng-mouseover="$ctrl.blurNavigationLinks($event)" ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_DONATE">
-        <div class="oppia-navbar-button-container">
-          <a href="/donate"
-             class="btn oppia-navbar-button nav-link oppia-navbar-hide-on-small-width oppia-transition-200">
-            <span class="oppia-navbar-tab-content" translate="I18N_TOPNAV_DONATE"></span>
+    <li>
+      <ul ng-if="$ctrl.windowIsNarrow" class="nav oppia-navbar-tabs-narrow">
+        <li><create-activity-button></create-activity-button></li>
+      </ul>
+    </li>
+    <li>
+      <ul ng-if="!$ctrl.windowIsNarrow" class="nav oppia-navbar-tabs">
+        <li ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_LIBRARY" class="nav-item oppia-clickable-navbar-element">
+          <a ng-mouseover="$ctrl.blurNavigationLinks($event)" class="nav-link oppia-navbar-tab oppia-navbar-tab-content" href="/library" translate="I18N_TOPNAV_LIBRARY"></a>
+        </li>
+        <li ng-class="{'open' : $ctrl.activeMenuName === 'aboutMenu'}" uib-dropdown ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_ABOUT" class="nav-item  oppia-navbar-clickable-dropdown protractor-test-about-oppia-list-item">
+          <a ng-keydown="$ctrl.onMenuKeypress($event, 'aboutMenu', {shiftTab: $ctrl.ACTION_CLOSE, enter: $ctrl.ACTION_OPEN})" class="nav-link oppia-navbar-tab dropdown-toggle"
```

Comment:
Can you format these lines to have each of the attributes on a new line? There's a lot going on here, so it would be easier to read.

Reply:
Done